tests/test_api.py

```diff
@@ -219,6 +219,76 @@ def test_create_prod_build_build_request(self, osbs, inner_template,
                                           customize_conf=customize_conf)
         assert isinstance(response, BuildResponse)
 
+    def test_create_prod_build_remove_koji_task_id(self):
```

Comment:
Can we verify that it behaves gracefully when there is already no "koji-task-id" annotation?